recipes/bettercam/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "bettercam" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/bettercam-{{ version }}.tar.gz
+  sha256: 122cc0e905cb40be8f2a26e46ee587bae40c10eb9c23201370a54e68b84a0940
+
+build:
+  skip: true  # [not win]
```

Comment:
This looks like it ca be noarch python

Reply:
due to lack of non-windows build in `comtypes`(which is dependency of `bettercam`), the build fails. Also since the `bettercam` utilizes Windows API, installing this on non-windows OS has no effects.

Reply:
If you do not install comtypes, running `import bettercam` will result in an error because it contains a line that imports comtypes. In other words, you can't even install the package without it, even if it doesn't work.

Reply:
Make this a Windows only noarch. See https://github.com/conda-forge/dxcam-feedstock/blob/main/recipe/meta.yaml for an example.